### ISSUE TITLE
Leave Suggestions for Settings Here!

### ISSUE THREAD

**Issue by Sheshank-s** [GreenBayRules/quark-editor#1]
Suggestions:
- @WorldLanguages Beautify button. My comments: Possible, and will do.

**Comment by Sheshank-s** [comment]
\cc @VutonCodes @csf30816 @I'llupdatethiswhenithinkofmorepeopletobother

**Comment by kerrtravers** [comment]
I think it would be really cool if you could adjust the section sizes. This would mean that you could have a bigger workspace for your code or a bigger preview of your site.

**Comment by Sheshank-s** [comment]
Good idea! I couldn't really figure out how to do this with a scroll like how scratchpad.io does it, so I might make it an option in settings

**Comment by Sheshank-s** [comment]
@vutondesign I made something similar in settings, check it out!

**Comment by kerrtravers** [comment]
Cool! 👍

**Comment by Sheshank-s** [comment]
@vutondesign Got any other feedback?

**Comment by kerrtravers** [comment]
Hmm, Not at the moment.

**Comment by SheepTester** [comment]
It might be nice to have an option to disable the automatic preview (and replace it with a run button/keyboard shortcut maybe)

**Comment by Sheshank-s** [comment]
@SheepTester You can do that in the settings. But I might make a key shortcut for it too

**Comment by BenedictKnows** [comment]
I know this is supper lazy, But maybe a reset button xP

**Comment by Sheshank-s** [comment]
CTRL+A then Delete @WAKOgaming

**Comment by Sheshank-s** [comment]
@csf30816

**Comment by Sheshank-s** [comment]
@csf30816 what is your email? ill add you to a slack
